### PR TITLE
Aashirya

### PR DESCRIPTION
**Pull request by Aashirya1995**

### CODE REVIEW

File: docker-compose.test.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 sut:
-  build:
-  command: ./run_tests.sh
\ No newline at end of file
+  build: .
+  command: ./run-test.sh
```
Comment (mathyourlife) [inline docker-compose.test.yml]: Looks like your run-test.sh file is not marked as an executable file.
Try changing this line from

```bash
  command: ./run-test.sh
```

to

```bash
  command: bash ./run-test.sh
```

Comment (Aashirya1995) [reply docker-compose.test.yml]: Thanks, that was really helpfull.

Comment (mathyourlife) [reply docker-compose.test.yml]: :+1:

**Comment by mathyourlife** [comment]
:+1:

**Pull request merged**